nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/MemberAddressPicker.java

```diff
@@ -151,22 +189,21 @@ private void findAppropriateInterfaces() {
             while (interfaces.hasMoreElements()) {
                 NetworkInterface intf = interfaces.nextElement();
                 logger.log(Level.FINE, "Found Network Interface {0}", new Object[]{intf.getName()});
-                
-                if (intf.isUp() && !intf.isLoopback() && !intf.isVirtual() && !intf.getName().contains("docker0") &&!intf.getDisplayName().contains("Teredo") && intf.getInterfaceAddresses().size()>0) {
+
+                if (intf.isUp() && !intf.isLoopback() && !intf.isVirtual() && !intf.isPointToPoint() && !intf.getName().contains("docker0") &&
```

Comment:
Presumably _docker0_ is the default Linux interface name for the Docker network?
On Windows it appears to resolve to _host.docker.internal_, so I don't know if you need to account for that.

Reply:
@smillidge put that in. I have no idea